lms/djangoapps/grades/models.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from xmodule_django.models import BlockTypeKeyField, CourseKeyField, UsageKeyField
+
+class Grade(models.Model):
+    """
+    A django model tracking persistent grades at the subsection level.
+    """
+
+    class Meta(object):
+        index_together = [
+            ['user_id', 'course_id', 'content_type', 'is_valid'],
+            ['course_id', 'updated'],
+            ['updated'],
+        ]
+
+    # TODO: timestamp discussions - do we need all three of [created, updated, subtree_edited_date]?
+    created = models.DateTimeField('creation timestamp', auto_now_add=True)
+    updated = models.DateTimeField('latest updated timestamp', auto_now=True)
+    subtree_edited_date = models.DateTimeField('last content edit timestamp')
+
+    # TODO: is this the right way to store course edit version? make sure it matches up w/ what Nimisha's exposing in her PR
+    course_version = models.CharField('guid of latest course version', max_length=255)
+
+    user = models.ForeignKey(User)
```

Comment:
Just a warning that this is one of the tables in our system that is most likely to eventually be detached into a different data store, so you shouldn't build too much on the assumption that this foreign key exists. You do have to store the user ID of course, and I'm not asking you to remove this now. Just please be aware as you're building that this information may eventually have different pluggable backends -- and build your abstractions accordingly.